docs/articles_en/documentation/openvino-ir-format/operation-sets/operations-specifications/matrix/Inverse_14.rst

```diff
@@ -0,0 +1,135 @@
+.. {#openvino_docs_ops_matrix_Inverse_14}
+
+Inverse
+=======
+
+
+.. meta::
+  :description: Learn about Inverse-14 - a matrix operation that computes the inverse or adjoint of one or multiple input matrices.
+
+**Versioned name**: *Inverse-14*
+
+**Category**: *Matrix*
+
+**Short description**: *Inverse* operation computes either the inverse or adjoint (conjugate transposes) of one or more square invertible matrices.
+
+**Detailed description**: *Inverse* operation computes the inverse of a square matrix. The operation uses LU decomposition with partial pivoting to compute the inverses.
+
+The inverse matrix A^(-1) of a square matrix A is defined as:
+
+.. math::
+
+   A \cdot A^{-1} = A^{-1} \cdot A = I
+
+where **I** is the n-dimensional identity matrix.
+
+The inverse matrix exists if and only if the input matrix is invertible. In that case, the inverse is unique. If the matrix is not invertible however, the operation may raise an exception or return an undefined result.
+
+This operation can be used to compute the adjugate matrix instead of the inverse.
+
+The adjugate matrix adj(A) of a square matrix A is defined as:
+
+.. math::
+
+   adj(A) = det(A) \cdot A^{-1}
+
+where **A^{-1}** is the matrix inverse of A, and **det(A)** is the determinant of A.
+
+The adjugate matrix exists if and only if the inverse matrix exists.
+
+**Inputs**:
+
+* **1**: `input` - A tensor of shape [B1, B2, ..., Bn, ROW, COL] and type `T` representing the input square matrices. The inner-most 2 dimensions form square matrices and must be of the same size. **Required.**
```

Comment:
`B1, B2, ..., Bn` are the batch dimensions

Reply:
Described!